.gitignore

```diff
@@ -24,3 +24,4 @@ docs
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+package-lock.json
```

Comment:
Since the  `package-lock.json` file was already added, it needs to be deleted from the repo as well as adding it as ignored.

Reply:
done